dev/com.ibm.ws.microprofile.openapi/resources/com/ibm/ws/microprofile/openapi/resources/OpenAPI.nlsprops

```diff
@@ -1,25 +1,26 @@
-#CMVCPATHNAME N/A
-#COMPONENTPREFIX CWMMC
-#COMPONENTNAMEFOR CWMMC MicroProfile OpenAPI API 
-#NLS_ENCODING=UNICODE
-#NLS_MESSAGEFORMAT_NONE
-#ISMESSAGEFILE true
-# #########################################################################
-###############################################################################
-# Copyright (c) 2009, 2011 IBM Corporation and others.
+# Copyright (c) 2018 IBM Corporation and others.
 # All rights reserved. This program and the accompanying materials
 # are made available under the terms of the Eclipse Public License v1.0
 # which accompanies this distribution, and is available at
 # http://www.eclipse.org/legal/epl-v10.html
 #
 # Contributors:
 #     IBM Corporation - initial API and implementation
-###############################################################################
-# This file follows the WebSphere Message Guidelines.
-# For more information, visit: 
-# http://washome.austin.ibm.com/xwiki/bin/view/MessagesTeam/
-#
+# -------------------------------------------------------------------------------------------------
+#CMVCPATHNAME N/A
+#COMPONENTPREFIX CWWKO
+#COMPONENTNAMEFOR CWWKO MicroProfile OpenAPI API
+#ISMESSAGEFILE TRUE
+#NLS_MESSAGEFORMAT_VAR
+#NLS_ENCODING=UNICODE
+# -------------------------------------------------------------------------------------------------
+
+# Message prefix block: CWWKO1650 - CWWKO1699
+
+OPENAPI_DOCUMENT_VALIDATION_ERROR=CWWKO1650E: Validation of the Open API document produced the following error. Message: {0}, Location: {1}
```

Comment:
Suggestion to use "OpenAPI" instead of "Open API". I know it's not consistent everywhere, I have seen both as well. But at least the [OAS](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md) and [MP spec](https://github.com/eclipse/microprofile-open-api/blob/master/spec/src/main/asciidoc/microprofile-openapi-spec.adoc) uses "OpenAPI":

Reply:
There are other messages that refer to it as an "Open API document". Copied the text from one of those. I agree that we should make it consistent.